manuscript/chapter2.md

```diff
@@ -985,9 +989,9 @@ const isSearched = searchTerm => item =>
   item.title.toLowerCase().includes(searchTerm.toLowerCase());
 ~~~~~~~~
 
-One could argue which function is more readable. Personally I prefer the second one. The React ecosystem uses a lot of functional programming concepts. It happens often that you will use a function which returns a function (higher-order functions). In JavaScript ES6, you can express these more concisely with arrow functions.
+The React ecosystem uses a lot of functional programming concepts, often using functions that return high-order functions to pass information. JavaScript ES6 lets us express these even more concisely with arrow functions.
```

Comment:
should be "... often using functions that return functions (high-order functions) to pass ..."

Reply:
The phrase "function(high-order functions)" doesn't make sense, because a high-order function is a function, so the extra "function" outside the parenthesis seems pointless when you can just remove the parenthesis altogether and describe the thing being passed in 2 words instead of 3.